app/javascript/controllers/admin_days_limit_controller.js

```diff
@@ -0,0 +1,9 @@
+import { Controller } from '@hotwired/stimulus'
+
+export default class extends Controller {
+  static targets = ['daysLimit']
```

Comment:
This target doesn't seem to be used anywhere in the controller. If so, can we dump it? Am I misreading? 😵‍💫